zerver/openapi/python_examples.py

```diff
@@ -693,21 +693,31 @@ def send_message(client):
 
     return message_id
 
+@openapi_test_function("/messages/{message_id}/reactions:post")
 def add_reaction(client, message_id):
     # type: (Client, int) -> None
+    # {code_example|start}
+    # Add an emoji reaction
+    # (make sure that message_id below is set to the ID of the
+    # message you wish to add reaction to)
     request = {
         'message_id': str(message_id),
         'emoji_name': 'joy',
         'emoji_code': '1f602',
```

Comment:
I think this example should just send `emoji_name`; I'll also write a section for the docs explaining what `emoji_code` is needed for.